pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+### Pull Requests
+
+Please be sure you have read our documentation regarding contributing
+here: https://docs.pulpproject.org/index.html
+
+There you can read a Contribution Checklist and learn more about
+appropriate branches and rebasing among other things.
+
+Once you are ready to create a pull request:
+
+- [ ] If there is a bug for this issue, please title the pull request
+"<bug_id> - Short Message"
+- [ ] In the commit message or comment section for your pull request,
+please include a link to the issue.
+- [ ] If there is a Redmine issue associated with the commit, please
+add "closes #<issue number>" in your commit as this will set the
+issue to MODIFIED in the tracker upon merging.
```

Comment:
Not sure I would mention setting the issue to MODIFIED since upon merging it should automatically happen.

Reply:
It says that it will be done automatically, it's not an instruction, just information as to why the "closes #" is important.

Reply:
Ah yes, I misread it. 👍